donate/payments/tasks.py

```diff
@@ -17,6 +17,7 @@
     get_plan_id,
     get_merchant_account_id_for_card
 )
+from ..settings.languages import LANGUAGE_IDS
```

Comment:
you should never import settings as a file, because there is no guarantee you're loading the right one. Instead, ask django its current config:

```suggestion
from dangjo.conf import settings
```

and then you can safely extract anything from there:

```
LANGUAGE_IDS = settings.languages.LANGUAGE_IDS
```

Reply:
Hi @Pomax thank you for the catch and for the suggestion. 

One thing I did note was that referring to language id's through:

`settings.languages.LANGUAGE_IDS`
`settings.LANGUAGE_IDS`
`languages.LANGUAGE_IDS`

using the suggested method did not work and returned an error stating that "settings object has no attribute 'languages'".

Would defining the language_ids in the base.py file as the current PR is doing, and then referring to them as:
`LANGUAGE_IDS = settings.LANGUAGE_IDS` work, Or is there a downside in having to define language_ids in base.py?